sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -1230,16 +1230,10 @@ private[hive] object HiveClientImpl extends Logging {
 
   // Visible for testing.
   private[hive] val HIVE_METASTORE_GENERATED_PROPERTIES: Set[String] = Set(
```

Comment:
After check Hive1.2 and Hive2.3 again, I think we can just remove 3 properties so that we can reduce the scope of influence.

Reply:
Does this include all the properties in https://github.com/apache/spark/pull/28647/files#diff-b7094baa12601424a5d19cb930e3402fL1534 ?

Reply:
Yes, I checked all the properties and make sure hive just modify the three properties.